src/test-run/index.js

```diff
@@ -178,6 +179,9 @@ export default class TestRun extends Session {
         if (this.pendingRequest)
             this._resolvePendingRequest(command);
 
+        if (command.type === COMMAND_TYPE.wait)
```

Comment:
It's debatable. But I think we shouldn't since we not operate with page. The reason we reject actions with js errors is: 
a)to have early errors 
b)associate action with error (error may be a side effect of the action or action is performed on already broken page)

I think `wait`'s doesn't meet the second criteria, since we don't perform any actions. It's like with Hybrids we just perform observation (especially in case of `waitForElement`)